packages/fluentui/accessibility/src/behaviors/Tree/treeItemAsOptionBehavior.ts

```diff
@@ -0,0 +1,36 @@
+import * as _ from 'lodash';
+import { Accessibility } from '../../types';
+import { treeItemBehavior, TreeItemBehaviorProps } from './treeItemBehavior';
+import { treeTitleAsOptionBehavior } from './treeTitleAsOptionBehavior';
+
+/**
+ * @description
+ * Adds role 'option' to a non-leaf item.
+ * Adds 'aria-selected' to a non-leaf item if it is selectable, based on the 'selected' property
+ */
+export const treeItemAsOptionBehavior: Accessibility<TreeItemBehaviorProps> = props => {
+  const behavior = treeItemBehavior(props);
+
+  const definition = _.merge(behavior, {
```

Comment:
I think we need be careful here which aria attribute we will apply on the option role.
https://www.w3.org/TR/wai-aria-1.1/#option

Because looks like, "aria-expanded", "aria-level" is not applicable on role=option.